community/examples/ml-gke.yaml

```diff
@@ -19,6 +19,8 @@ vars:
   project_id:  ## Set GCP Project ID Here ##
   deployment_name: ml-01
   region: us-central1
```

Comment:
I suggest considering asia-southeast1-b in asia-southeast1 as a region with more capacity.